src/common/helpers/DeclarationIndexHelpers.ts

```diff
@@ -60,7 +60,7 @@ export function getAbsolutLibraryName(library: string, actualFilePath: string, r
     return '/' + toPosix(relative(
         rootPath,
         normalize(join(parse(actualFilePath).dir, library)),
-    )).replace(/[/]$/g, '');
+    )).replace(/\/$/, '');
```

Comment:
- Escaping is idiomatically done with backslashes (you usually do, btw)
- You seem to be putting the `g` flag everywhere, but 99% of the time you don't need it, and it entails more overhead / different return semantics / sticky behavior.